Fix a build error during module install

make[6]: *** No rule to make target '/tmp/buildd/linux-3.5.7/debian/linux-image-3.5.0-1mk-xenomai-686-pae/lib/firmware/./', needed by '/tmp/buildd/linux-3.5.7/debian/linux-image-3.5.0-1mk-xenomai-686-pae/lib/firmware/atmsar11.fw'.  Stop.
make[6]: *** Waiting for unfinished jobs....

https://bugzilla.kernel.org/show_bug.cgi?id=39192

--- a/scripts/Makefile.fwinst.fw_install	2012-10-12 20:48:25.000000000 +0000
+++ b/scripts/Makefile.fwinst	2014-05-13 15:27:52.647743463 +0000
@@ -27,7 +27,7 @@
 installed-mod-fw := $(addprefix $(INSTALL_FW_PATH)/,$(mod-fw))
 
 installed-fw := $(addprefix $(INSTALL_FW_PATH)/,$(fw-shipped-all))
-installed-fw-dirs := $(sort $(dir $(installed-fw))) $(INSTALL_FW_PATH)/.
+installed-fw-dirs := $(sort $(dir $(installed-fw))) $(INSTALL_FW_PATH)/./
 
 # Workaround for make < 3.81, where .SECONDEXPANSION doesn't work.
 PHONY += $(INSTALL_FW_PATH)/$$(%) install-all-dirs
